src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.Aot.targets

```diff
@@ -79,7 +85,11 @@ They run in a context of an inner build with a single $(RuntimeIdentifier).
         OutputType="Library"
         UseAotDataFile="false"
         UseLLVM="$(EnableLLVM)"
-        LLVMPath="$(_LLVMPath)">
+        LLVMPath="$(_LLVMPath)"
+        LdName="$(_LdName)"
+        LdFlags="$(_LdFlags)"
+        WorkingDirectory="$(MSBuildProjectDirectory)"
```

Comment:
Will this still work if the user has not ascii chars in its `Configuration` ? e.g `Débug` ?

Reply:
This just gets it on par with Xamarin.Android now, so it's going to be the same.

Right now, I would guess that it would break.

Reply:
ok as long as we are on par with classic this should be good.